junit-runtime/src/main/scala/com/novocode/junit/OutputCapture.scala

```diff
@@ -0,0 +1,47 @@
+package com.novocode.junit
+
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+object OutputCapture {
+  def start(): OutputCapture = new OutputCapture()
+
+  private def getScalaOut(): PrintStream = {
+    try {
+      scala.Console.out
+    } catch {
+      case t: Throwable => null
+    }
+  }
+
+  private def setScalaOut(p: PrintStream): Boolean = {
+    try {
+      scala.Console.setOut(p)
+      true
+    } catch {
+      case t: Throwable => false
+    }
+  }
+}
+
+final class OutputCapture private[OutputCapture]() {
+  private val originalOut = System.out
+  private val originalScalaOut = OutputCapture.getScalaOut()
+  private val buffer = new ByteArrayOutputStream()
+  private val prBuffer = new PrintStream(buffer, true)
+  private val scalaOutSet: Boolean = OutputCapture.setScalaOut(prBuffer)
```

Comment:
It is extremely strange, but it is how the sbt JUnit interface does it on the JVM. I believe that it should behave the same way on Scala.js JUnit sbt interface. Even if the way to handle the output is a horrible mess.

https://github.com/sbt/junit-interface/blob/master/src/main/java/com/novocode/junit/OutputCapture.java